oaknational/Oak-Web-Application
# fix: show line breaks in curriculum explainer

## Description

-- Show line breaks for curriculum explainer statements on Overview tab in curriculum view

## How to test

1. Go to {owa_deployment_url}/teachers/curriculum/maths-secondary/overview
3. You should see line breaks in the explainer statement

## Screenshots

How it used to look (delete if n/a):
<img width="1227" alt="Screenshot 2023-10-11 at 11 10 57" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/6ad0fa78-797c-4b29-b191-48f51af7358b">

How it should now look:
<img width="1225" alt="Screenshot 2023-10-11 at 11 11 35" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/a9f85bf0-f13c-4389-918b-1595094d7f65">


## Checklist
